spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerTemplate.java

```diff
@@ -152,6 +153,17 @@ public long executeDmlStatement(Statement statement) {
 		return rowsAffected;
 	}
 
+	@Override
+	public long executePartitionedDmlStatement(Statement statement) {
+		Assert.notNull(statement, "A non-null statement is required.");
+		maybeEmitEvent(new BeforeExecuteDmlEvent(statement));
+		long rowsAffected = doWithOrWithoutTransactionContext(x -> {
+			throw new SpannerDataException("Cannot execute partitioned DML in a transaction.");
+		}, () -> this.databaseClientProvider.get().executePartitionedUpdate(statement));
```

Comment:
is it properly formatted?

Reply:
yes I did this manually and then did auto reformat which did not change this at all. alas, the checkstyle passes.